scripts/anvil_tools/README.md

```diff
@@ -44,7 +44,7 @@
     Locally
         `./file_exists_checker EXCEL_FILE COLUMN_NUMBER_TO_CHECK`
     Docker
-        `docker run --rm -it -v "$HOME"/.config:/.config -v "$HOME"/local_data_directory/:/data broadinstitute/horsefish:latest bash -c "cd data; ./file_exists_checker EXCEL_FILE COLUMN_NUMBER_TO_CHECK"`
+        `docker run --rm -it -v "$HOME"/.config:/.config -v "$HOME"/local_data_directory/:/data broadinstitute/horsefish:latest bash -c "export CLOUDSDK_PYTHON=python2.7; cd data; ./../scripts/anvil_tools/file_exists_checker.sh EXCEL_FILE COLUMN_NUMBER_TO_CHECK; unset CLOUDSDK_PYTHON"`
```

Comment:
noooooo python 2 nooooooo - but seriously, can we please please please not use python 2?

Reply:
oh is this outdated now?